velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.cpp

```diff
@@ -34,12 +35,28 @@ std::shared_ptr<FileSystem> abfsFileSystemGenerator(
   });
   return filesystem;
 }
+
+std::unique_ptr<velox::dwio::common::FileSink> abfsWriteFileSinkGenerator(
+    const std::string& fileURI,
+    const velox::dwio::common::FileSink::Options& options) {
+  if (isAbfsFile(fileURI)) {
```

Comment:
Do we need to make the ABS register function static, similar to the HDFS register function  [here](https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/storage_adapters/hdfs/RegisterHdfsFileSystem.cpp#L76), to avoid creating multiple instances?

Reply:
I think this is by design behavior, considering the token may expire, we need to create new instance with new tokens.